### ISSUE TITLE
refine registration task

### ISSUE THREAD

**Issue by qigongFDA** [DIDSR/eeDAP#81]
We want to create one task for refine registration. For one slide registered in previous study, user can skip whole global registration. Before evaluate the study ROI on the slide, user should use this task to refine registration. 

The ROI of this task is predefined in input file. During the task, stage should automatically move to the ROI or surrounding area. User can use any magnification for searching and moves stage to let the center of WSI is in the center of eyepiece, then uses 40x to refine registration.

**Comment by qigongFDA** [comment]
User might need to use low magnification WSI to search the ROI, and high magnification WSI to do registration. We the to show both low and high magnifications ROI. There are two plans:

Plan A:  Add a "switch button" in task panel. It can switch the WSI ROI between 10x and 40x.
Plan B: In eeDAP GUI, only show 40x ROI, and open imagescope with ROI annotation. 

Plan B gives user more flexibility, user can choose WSI magnification based on the lens. But the WSI image has to be in the format, which can be read by imagescope.

**Comment by brandon-gallas** [comment]
OK. How about this?
Once you have the task working, can we call that task from the GUI instead of including it in the input file?
I still think I like a GUI button "New Slide" or "Needs Registration" or "Register".
When you click this button, it takes you to the task you are talking about.
This task needs the location to do the registration.
We can get that knowing the slide ... We can use the location of the first registration anchor.

**Comment by brandon-gallas** [comment]
I don't think we should involve imagescope.
I like plan A. I'm happy to extract registration images on the fly (not extract ahead of time).

**Comment by qigongFDA** [comment]
![refineregistration](https://user-images.githubusercontent.com/12702285/44930220-9b917400-ad2b-11e8-80fb-a47c7cc59a7d.PNG)
Here is a flowchart for this feature.
![refineregistration](https://user-images.githubusercontent.com/12702285/44930245-a946f980-ad2b-11e8-8662-f2ac4e32343a.PNG)
[RefineRegistration.pptx](https://github.com/DIDSR/eeDAP/files/2341178/RefineRegistration.pptx)

**Comment by brandon-gallas** [comment]
Looks good. Let me know progress.
BTW, I think this "task" can be integrated into other tasks ... as an option or always available.

**Comment by qigongFDA** [comment]
Yes. The "Refine_Register" button is in "Control Panel". This feature should be available for all original tasks.

**Comment by qigongFDA** [comment]
1. New flowchart 
![flowchart](https://user-images.githubusercontent.com/12702285/45244506-1deadc80-b2c6-11e8-84c1-56a5207db108.PNG)

2. Variable and procedure path 
![variablepath](https://user-images.githubusercontent.com/12702285/45244516-2216fa00-b2c6-11e8-992f-22ee4e1b086e.PNG)
[RefineRegistration.pptx](https://github.com/DIDSR/eeDAP/files/2362609/RefineRegistration.pptx)
